r2/r2/lib/jsontemplates.py

```diff
@@ -283,6 +283,7 @@ class SubredditJsonTemplate(ThingJsonTemplate):
         collapse_deleted_comments="collapse_deleted_comments",
         comment_score_hide_mins="comment_score_hide_mins",
         # community_rules="community_rules",
+        creator="author",
```

Comment:
The word 'creator' was used for the same reason the terminology on account suspensions was changed in the json response; publicly, it's the subreddit's creator, not the subreddit's author.